config/storage.yml

```diff
@@ -6,13 +6,13 @@ local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
 
-# Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
-# amazon:
-#   service: S3
-#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
-#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
-#   region: us-east-1
-#   bucket: your_own_bucket
+# AWS secrets
+amazon:
+  service: S3
+  access_key_id: <%= ENV.fetch('AWS_ACCESS_KEY_ID', nil)} %>
+  secret_access_key: <%= ENV.fetch('AWS_ACCESS_SECRET_KEY_ID', nil) %>
```

Comment:
Is this configuration required for dev? Will it work locally on a machine without these keys defined?

Reply:
This configuration is currently running for the production, not for dev. AWS conf won't work locally. Files storage locally in the file sistem

Reply:
Do you mean that this file `config/storage.yml` would not even try to be loaded in dev? Would you happen to have proof of that?

(It's not a blocker for this PR anymore, but I think it would not hurt if you check it for your self)